posthog/session_recordings/session_recording_api.py

```diff
@@ -756,23 +766,13 @@ def list_recordings(
             filter = filter.shallow_clone({SESSION_RECORDINGS_FILTER_IDS: remaining_session_ids})
 
     if (all_session_ids and filter.session_ids) or not all_session_ids:
-        has_hog_ql_filtering = request.GET.get("hog_ql_filtering", "false") == "true"
```

Comment:
Can remove the corresponding value from the [sessionRecordingsPlaylistLogic](https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts#L290)